src/routes/welcome.ts

```diff
@@ -22,12 +23,40 @@ export const createWelcomeRoute: RouteCreator =
       ).data.logout_url || ""
 
     res.render("welcome", {
-      session: session
-        ? JSON.stringify(session, null, 2)
-        : `No valid Ory Session was found.
-Please sign in to receive one.`,
-      hasSession: Boolean(session),
-      logoutUrl,
+      layout: "welcome",
+      nav: navigationMenu(session, logoutUrl, "welcome"),
+      concepts: [
+        CardGradient({
+          heading: "Getting Started",
+          content:
+            "Jump start your project and complete the quickstart tutorial to get a broader overview of Ory Cloud.",
+          action: "/",
+        }),
+        CardGradient({
+          heading: "User flows",
+          content:
+            "Implement flows that users perform themselves as opposed to administrative intervention.",
+          action: "/",
+        }),
+        CardGradient({
+          heading: "Identities 101",
+          content:
+            "Every identity can have its own model - get to know the ins and outs of Identity Schemas.",
+          action: "/",
+        }),
+        CardGradient({
+          heading: "Sessions",
+          content:
+            "Define Text here Jump start your project and complete the quickstart tutorial to get a broader",
+          action: "/",
+        }),
+        CardGradient({
+          heading: "Custom UI",
+          content:
+            "Implementing these pages in your language and framework of choice is straightforward using our SDKs.",
+          action: "/",
+        }),
+      ].join("\n"),
```

Comment:
How does this work? Does `CardGradient` just return a string?

Reply:
Yes, and handlebars injects the array of strings into the template as just string which is rendered as HTML.

Reply:
I see, should `\n` not be `<br />` then?

Reply:
no, since this is just a simple array that is concatenating it. we can simply do a `.join("")` as well. but the html won't look so pretty on render :)

Reply:
Ah, alright. Kinda weird to see a `\n` in combination with HTML, though ^^. But LGTM then.